data/mods/CRT_EXPANSION/monsters/crt_monster.json

```
@@ -293,5 +293,39 @@
       "PUSH_MON",
       "FILTHY"
     ]
+  },
+  {
+    "id": "mon_spider_disease",
+    "type": "MONSTER",
+    "name": { "str": "shimmering spider" },
+    "description": "A dog-sized spider, the air around it has a faint shimmer like light on strands of webbing.",
```

```suggestion
    "description": "A dog-sized spider, the air around it has a faint shimmer of light, possibly reflected from tiny strands of webbing.",
```
"like" implies it's _not_ from strands of webbing.